src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,14 @@
+.page__body {
+  margin: 0;
+}
+
+h1,
+h2,
+h3,
+p,
+a,
```

Comment:
`a` doesn't have `padding` and `margin` as a default values, it's an `inline` element